src/main/java/berlin/yuna/nano/services/metric/logic/MetricService.java

```diff
@@ -36,13 +40,11 @@ public MetricService() {
     @Override
     public void start(final Supplier<Context> contextSupplier) {
         isReady.set(false, true, run -> updateSystemMetrics());
-        //addlistener for event
-
-        // metrics.prometheus(); this will the response body
-        // GET call
-        // http://localhost:8080/metrics thinks of good word
-        // status code 200
-
+        Optional<String> basePath = contextSupplier.get().getOpt(String.class, Config.CONFIG_METRIC_SERVICE_BASE_PATH.id());
```

Comment:
move this inside the `isReady.set` beckets as the ready state is set to true before the paths are evaluated